clients/ts/FunctionalTests/func.jest.config.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+module.exports = {
+    transformIgnorePatterns: [
+        ".*/node_modules/(?!@aspnet)"
```

Comment:
Since this is a JS file, not a JSON file, can you put a comment in? Something like "We reference the ESM output directly in our tests and need it to run through the TS compiler in order to work in NodeJS"